ulauncher/modes/extensions/extension_controller.py

```diff
@@ -78,21 +78,32 @@ def __init__(self, ext_id: str, path: str | None = None) -> None:
 
     @classmethod
     def create(cls, ext_id: str, path: str | None = None) -> ExtensionController:
-        cached_controller = controller_cache.get(ext_id)
-        if cached_controller:
-            return cached_controller
+        controller = controller_registry.get(ext_id)
+        if controller:
+            return controller
```

Comment:
The name change from cached_controller to controller is already in main now, but with a walrus operator (I think it made more sense with the walrus. That's the conflict with main.

I didn't rename controller_cache to controller_registry though. I'm not sure I understand that change, but I also don't have any strong opinions on the name. It can be either.